fdbcli/BlobKeyCommand.actor.cpp

```diff
@@ -153,17 +162,15 @@ ACTOR Future<bool> blobKeyCommandActor(Database localDb,
 		return false;
 	}
 
-	ASSERT(tokens[1] == "history"_sr);
-
-	Key key;
-	Optional<Version> version;
-
+	Optional<Reference<Tenant>> tenant;
 	if (tenantEntry.present()) {
-		key = tokens[2].withPrefix(tenantEntry.get().prefix);
-	} else {
-		key = tokens[2];
+		tenant = Reference<Tenant>(new Tenant(tenantEntry.get().id));
```

Comment:
Yeah fdbcli has ITenant but not Tenant, and this seemed like the easiest way to get a Tenant object

Reply:
In theory all fdbcli commands are meant to use `IClientApi` (`ITransaction`, `IDatabase`, `ITenant`) so that they can work with the MVC. Not all of the commands are fully able to do this currently, though.